src/DETHRACE/common/network.c

```diff
@@ -45,7 +45,7 @@ int gRace_only_flags[33] = {
 };
 int gJoin_list_mode;
 tNet_game_player_info gNew_net_players[6];
-tGuaranteed_message gGuarantee_list[150];
+tGuaranteed_message gGuarantee_list[100]; // DOS debug symbols has this as [150]
```

Comment:
Perhaps they increased it for carmageddon 2.
I found functions in carmageddon 2 that are present in these debug symbols, but not in the carmageddon 1 executables.